### ISSUE TITLE
Show dynamic linked libraries for Darwin

### ISSUE THREAD

**Issue by jamesodhunt** [jamesodhunt/procenv#23]
Find a way to support `--libs` on Darwin.

The documented method is to use `_dyld_image_count()` to return an index of linked libraries and then for each index call `_dyld_get_image_name(i)`. However, whilst `otool -L $binary"` generally returns just `/usr/lib/libSystem.B.dylib` for a simple binary:

```bash
$ otool -L bin/test_list_shared_libs_darwin
bin/test_list_shared_libs_darwin:
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.100.5)
```

... Using the dyld calls returns the following list for that same simple binary (one of which is `/usr/lib/libSystem.B.dylib`):

```
/usr/lib/libSystem.B.dylib
/usr/lib/libc++.1.dylib
/usr/lib/libc++abi.dylib
/usr/lib/liboah.dylib
/usr/lib/libobjc.A.dylib
/usr/lib/system/libcache.dylib
/usr/lib/system/libcommonCrypto.dylib
/usr/lib/system/libcompiler_rt.dylib
/usr/lib/system/libcopyfile.dylib
/usr/lib/system/libcorecrypto.dylib
/usr/lib/system/libdispatch.dylib
/usr/lib/system/libdyld.dylib
/usr/lib/system/libkeymgr.dylib
/usr/lib/system/liblaunch.dylib
/usr/lib/system/libmacho.dylib
/usr/lib/system/libquarantine.dylib
/usr/lib/system/libremovefile.dylib
/usr/lib/system/libsystem_asl.dylib
/usr/lib/system/libsystem_blocks.dylib
/usr/lib/system/libsystem_c.dylib
/usr/lib/system/libsystem_collections.dylib
/usr/lib/system/libsystem_configuration.dylib
/usr/lib/system/libsystem_containermanager.dylib
/usr/lib/system/libsystem_coreservices.dylib
/usr/lib/system/libsystem_darwin.dylib
/usr/lib/system/libsystem_dnssd.dylib
/usr/lib/system/libsystem_featureflags.dylib
/usr/lib/system/libsystem_info.dylib
/usr/lib/system/libsystem_kernel.dylib
/usr/lib/system/libsystem_m.dylib
/usr/lib/system/libsystem_malloc.dylib
/usr/lib/system/libsystem_networkextension.dylib
/usr/lib/system/libsystem_notify.dylib
/usr/lib/system/libsystem_platform.dylib
/usr/lib/system/libsystem_product_info_filter.dylib
/usr/lib/system/libsystem_pthread.dylib
/usr/lib/system/libsystem_sandbox.dylib
/usr/lib/system/libsystem_secinit.dylib
/usr/lib/system/libsystem_symptoms.dylib
/usr/lib/system/libsystem_trace.dylib
/usr/lib/system/libunwind.dylib
/usr/lib/system/libxpc.dylib
```

**Comment by jamesodhunt** [comment]
Ah! Worked it out: macho load commands! ;) :muscle: :smile:

**Comment by jamesodhunt** [comment]
Fixed by 02d5dbc96aba82c5b0c0307904d85ae0306fa0cc.
